pkgs/by-name/wt/wtf/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  cmake,
+  ninja,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wtf";
+  version = "0.5.7";
+
+  src = fetchFromGitHub {
+    owner = "0vercl0k";
+    repo = "wtf";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-lfPx9RQEW457VQkDbvu/D9EFZrdNLz2ToQ9dfa7+tzY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+  ];
+
+  sourceRoot = "source/src";
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
```

Isn't the default build type release (worth checking)? Otherwise let use
```
```suggestion
  cmakeFlags = [
    (lib.cmakeFeature "CMAKE_BUILD_TYPE" "Release")
  ];
```

---

Good question. I take if from the [build script](https://github.com/0vercl0k/wtf/blob/main/src/build/build-release.sh), used in the [CI build](https://github.com/0vercl0k/wtf/blob/main/.github/workflows/wtf.yml).

---

I meant on nixpkgs cmake builder